mmcv/runner/fp16_utils.py

```diff
@@ -82,6 +91,7 @@ def new_func(*args, **kwargs):
                                 'method of nn.Module')
             if not (hasattr(args[0], 'fp16_enabled') and args[0].fp16_enabled):
```

Comment:
I may suggest that we can enforce the input tensor to be float32 in this case, so that `auto_fp16` can be used in intermediate layers in truly mixed-precision training.

Reply:
leave it later